SPECS-EXTENDED/python-trio/python-trio.spec

```
@@ -27,12 +27,12 @@ right.}
 
 Name:           python-%{pkgname}
 Version:        0.16.0
-Release:        2%{?dist}
+Release:        3%{?dist}
 Summary:        A friendly Python library for async concurrency and I/O
 License:        MIT or ASL 2.0
 URL:            https://github.com/python-trio/trio
 #Source0:       https://files.pythonhosted.org/packages/source/t/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
-Source0:        https://files.pythonhosted.org/packages/source/t/%{pypi_name}/%{name}-%{version}.tar.gz
+Source0:        https://github.com/%{pypi_name}/%{name}/archive/v%{version}.tar.gz
```

I don't think the new link works (there's no `pypi_name`) but I've noticed this will work:
```suggestion
Source0:        https://files.pythonhosted.org/packages/source/t/%{pkgname}/%{pkgname}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

The signature of the file under that URL is the same as what we've got, so this should work well.

---

Here's the updated url: https://files.pythonhosted.org/packages/source/t/%{pypi_name}/%{pypi_name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz

After learning from the conversation here, I realized that the original link is actually good and the right macros are used.